src/store/ducks/opensrp/practitioners/index.ts

```diff
@@ -69,11 +70,14 @@ export type PractitionersActionTypes =
 
 /** Fetch practitioners action creator
  * @param {Practitioner []} practitionersList - practitioners array to add to store
+ * @param {boolean} overwrite - whether to replace the records in store for practitioners
  * @return {FetchPractitionersAction} - an action to add practitioners to redux store
  */
 export const fetchPractitioners = (
-  practitionersList: Practitioner[] = []
+  practitionersList: Practitioner[] = [],
+  overwrite: boolean = false
```

Comment:
@p-netm I know I said `false`, but now I'm wondering which case is more common for `overwrite`? If `overwrite` is set to `true` more often then `false` (which it looks like maybe it is?), then perhaps the default should be `true`. I'll leave that decision to you :)

Reply:
Well i would choose false since most of the other fetch action creators outside team assignment append as opposed to overwrite newly fetched records to store.